server/data/incentivesApi.test.ts

```diff
@@ -26,7 +26,9 @@ describe('IncentiveApi', () => {
       const locationId = 'ABC-1'
       const apiResponse = getTestIncentivesLocationSummary({ prisonId, locationId })
       incentivesApi
-        .get(`/incentives-summary/prison/${prisonId}/location/${locationId}`)
+        .get(
+          `/incentives-summary/prison/${prisonId}/location/${locationId}?sortBy=DAYS_SINCE_LAST_REVIEW&sortDirection=DESC`
```

Comment:
Sample sort example, will replace with params

Reply:
I left the sorting out as the default seems to be sorting by name (both in terms of Incentives API and in terms of requirements apparently) - but adding this here would mean the default ordering would now be wrong.